src/containers/quiz/quiz-question/Question.tsx

```diff
@@ -135,7 +135,7 @@ const QuizQuestion: FC<QuizQuestionProps> = ({
 
   const answersBlock = isOpenAnswer ? (
     <Answer
-      isCorrect
+      isCorrect={isCorrect}
       isEditable={isEditable}
```

Comment:
Use the shortest form for boolean props
```suggestion
      isEditable
```

Reply:
Same as in previous comment, sorry)